department-of-veterans-affairs/caseflow
# Hide mail team tasks from task snapshot

Resolves #10322 

### Description
Hides mail tasks assigned to the mail team from task snapshot

|Before|After|
|---|---|
|![Screen Shot 2019-10-01 at 4 12 47 PM](https://user-images.githubusercontent.com/45575454/65998182-cd15cb00-e468-11e9-8c66-bb04576b1d31.png)|![Screen Shot 2019-10-01 at 4 12 37 PM](https://user-images.githubusercontent.com/45575454/65998195-d0a95200-e468-11e9-84ef-32c970988c17.png)|


### Acceptance Criteria
- [ ] `MailTask`s assigned to the mail team do not appear in task snapshot

### Testing Plan
1. Switch users to the jolly postman
2. Navigate to /organizations/mail?tab=assigned
3. Click on the link for any cae
4. Ensure there are no mail tasks assigned to the mail team listed in task snapshot
